test/Language/Fortran/Parser/Fortran77/IncludeSpec.hs

```diff
@@ -32,10 +33,11 @@ spec =
 
         -- the expansion returns the span in the included file
         -- it should return the span at the inclusion
-        st2Span = makeSrcR (6,7,1,"foo.f") (14,15,1,"foo.f")
-        declSpan = makeSrcR (6,7,1,"foo.f") (14,15,1,"foo.f")
-        typeSpan = makeSrcR (6,7,1,"foo.f") (12,13,1,"foo.f")
-        blockSpan = makeSrcR (14,15,1,"foo.f") (14,15,1,"foo.f")
+        foo = head incs </> "foo.f"
```

Comment:
perhaps bit clearer without `head` by changing `incs` to one `inc :: FilePath` and wrapping it when parsing like `iParser [inc] source`?